packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -1161,7 +1161,8 @@ export class RemoteMediaManager extends EventsScope {
     remoteMedia.stop();
 
     delete this.media.video.memberPanes[paneId];
-    delete this.currentLayout.memberVideoPanes?.[paneId];
+    // TODO: to delete memberVideoPanes using paneId it needs to be a number
```

Comment:
I put **todo** here because types for `paneId` does not match and need to be changed properly

Reply:
@marcin-bazyl  sure